### PR TITLE
Print more details when flashing

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

File: sparse/boot/flash.sh [hunk 1]
```diff
@@ -194,7 +198,7 @@ if [ -z ${BLOB_BIN_PATH} ]; then
 fi
 
 BLOBS=""
-for b in $(ls -1 ${BLOB_BIN_PATH}/*_{v16,v17}_nile.img 2>/dev/null); do
+for b in $(ls -1 ${BLOB_BIN_PATH}*_{v16,v17}_nile.img 2>/dev/null); do
```
Comment (sledges) [inline sparse/boot/flash.sh:201]: This won't work if path is overridden without trailing slash, e.g. `export BLOB_BIN_PATH=~ ./flash.sh`

Comment (mkosola) [reply sparse/boot/flash.sh:201]: Ah, will fix to different place

Comment (sledges) [reply sparse/boot/flash.sh:201]: Why to different? `${BLOB_BIN_PATH}*_{v16,v17}_nile.img` now looks like a filename :)

Comment (mkosola) [reply sparse/boot/flash.sh]: Well if you flash normally, it will print .//SW... to blob name.

File: sparse/boot/flash.sh [hunk 2]
```diff
@@ -143,7 +147,7 @@ if [ "$($FASTBOOTCMD getvar secure 2>&1 | head -n1 | cut -d ' ' -f2 )" == "yes"
 fi
 
 if [ -z ${BINARY_PATH} ]; then
-  BINARY_PATH=./
+  BINARY_PATH=.
```
Comment (sledges) [inline sparse/boot/flash.sh:150]: This variable is not used anywhere

Comment (mkosola) [reply sparse/boot/flash.sh]: yes, wrong variable...

**Pull request merged**